cpp/modmesh/pilot/R3DWidget.cpp

```diff
@@ -84,8 +84,50 @@ void R3DWidget::updateWorld(std::shared_ptr<WorldFp64> const & world)
             child->deleteLater();
         }
     }
-    new RVertices(world, m_scene);
-    new RLines(world, m_scene);
+
+    // Create standalone Qt objects first
+    auto vertices = std::make_unique<RVertices>(world, nullptr);
```

Comment:
Create a QObject without assigning parent object at first.

Reply:
Why don't you let `vertices` and `lines` to set their parents (`m_scene`) here?